### DISCUSSION TITLE
Need helping setting up live stream

### DISCUSSION THREAD

**Discussion by FearlessNite345** [tsightler/ring-mqtt#623] [Support Q&A]
So im newer to this stuff and i want to get the live stream working for Ring cameras around my house but idk how to set that part up i do have the ring-mqtt working as i can easily add the snapshots but idk how to add a stream

**Comment by tsightler** [comment]
It's not clear what could be typed here that is not already covered in the fairly extensive [video streaming documentation](https://github.com/tsightler/ring-mqtt/wiki/Video-Streaming), which includes step-by-step instructions for adding streaming cameras to Home Assistant.  Have you attempted to follow it?  If so, perhaps you could describe what specifically you are having problems with and that would inform what assistance can be provided.

**Reply by FearlessNite345** [reply]
first things first im struggling to find what the "camera_id" is and where to locate it and second i didnt see that documentation so i will have a look at it as well

**Reply by tsightler** [reply]
The linked documentation should help you get the camera_id.  Usually, the trickiest part is the snapshot URL, which is difficult to get right for many new users that don't understand Home Assistants templating engine syntax (really Jinja2 syntax), but most users seem to manage to get there with a little trial and error.  If you get stuck with something, feel free to ask.
